File: data/StarCoder2/readable_format_HPEC/fortran/file037.f90
program DRB027_taskdependmissing_orig_yes
    use omp_lib
    implicit none

    integer :: i
    i=0

    !$omp parallel
    !$omp single
    !$omp task
    i = 1
    !$omp end task
    !$omp task
    i = 2
    !$omp end task
    !$omp end single
    !$omp end parallel

    print 100, i
    100 format ("i=",i3)

end program

File: data/llama3_8B_ins_results/readable_format_HPEC/cpp/file037.cpp
#include <iostream>
#include <omp.h>

int main() {
    int i = 0;

    #pragma omp parallel
    {
        #pragma omp single
        {
            #pragma omp task
            i = 1;
            #pragma omp task
            i = 2;
        }
    }

    std::cout << "i = " << i << std::endl;

    return 0;
}
